references/product-recommendations-api/set_env_variables_my_apigeex.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
```

Comment:
people could just use the `set_env_variables.sh` for the pipeline we could use env vars and set the rest in the pipeline itself.

Reply:
OK, removed